src/main/java/com/amplitude/api/AmplitudeClient.java

```diff
@@ -1795,7 +1806,14 @@ public void groupIdentify(String groupType, Object groupName, Identify groupIden
     }
 
     public void groupIdentify(String groupType, Object groupName, Identify groupIdentify, boolean outOfSession) {
-        groupIdentify(groupType, groupName, groupIdentify, false, null);
+        groupIdentify(groupType, groupName, groupIdentify, outOfSession, null);
+    }
+
+    public void groupIdentify(String groupType, Object groupName, JSONObject groupProperties, boolean outOfSession, MiddlewareExtra extra) {
```

Comment:
nit: Seems we don't have docs for any groupIdentify method. Not an issue for this PR though. Would be ideal if we can have some.